sources/assets/zsh/aliases.d/smbmap

```diff
@@ -0,0 +1 @@
+alias smbmap='cd /opt/tools/smbmap/ && echo "[*] changed dir : cd /opt/tools/smbmap/" && ./venv/bin/python3 smbmap/smbmap.py'
```

Comment:
Do we need to change directory? Can't we do something like `alias smbmap='/opt/tools/smbmap/venv/bin/python3 /opt/tools/smbmap/smbmap/smbmap.py'`
Also, why doing the venv manually while a pipx install should work? (`python3 -m pipx install git+https://github.com/ShawnDEvans/smbmap`)

Reply:
A local patch is applied and I don't think this is possible with `pipx`
```    
cp -v /root/sources/assets/patches/smbmap.patch smbmap.patch
git apply --verbose smbmap.patch
```

Reply:
Okay understood.
You can apply the patch then `python3 -m pipx install /path/to/smbmap`

Reply:
Does it work ?

Reply:
This was marked as resolved without actually changing code or answering the question, re-opening
Isn't possible to have `smbmap` installed with `pipx` and thus avoid the directory change in the alias?